scripts/local-cluster/README.md

```diff
@@ -17,4 +17,24 @@ Provided functionality includes:
 1. Download specified release versions of upstream Istio.
 1. Creating a k3s cluster with the 3scale adapter installed alongside required Istio version.
 1. Deploy a sample application into the service mesh. Currently [httpbin](https://httpbin.org) is supported with an opinionated `VirtualService` configuration.
-1. Clean-up of k3s cluster.
\ No newline at end of file
+1. Clean-up of k3s cluster.
+
+#### Troubleshooting
+
+##### DNS
+
+So you are having DNS issues?
+
+If your Pods cannot resolve external hostnames, check the logs from the `coredns` Pod in `kube-system` namespace.
+It is likely you will see i/o timeouts for services running on port 53 pointing some DNS issue in your environment.
+
+The following command will patch the appropriate ConfigMap. Replace `8.8.8.8` with your own nameservers' ip address:
+
+```yaml
+kubectl get cm coredns -n kube-system -o yaml | sed -e 's/proxy . \/etc\/resolv.conf/proxy . 8.8.8.8/g' | kubectl apply -f -
```

Comment:
this probably lacks the kube config param or env var, same thing below (maybe just export it once for the session as an env var and be done with it)

Reply:
solved above